haskell/nixpkgs.bzl

```diff
@@ -96,7 +96,11 @@ _ghc_nixpkgs_haskell_toolchain = repository_rule(
         "haddock_flags": attr.string_list(),
         "repl_ghci_args": attr.string_list(),
         "locale_archive": attr.string(),
-        "_nixpkgs_ghc": attr.label(default = "@io_tweag_rules_haskell_ghc_nixpkgs//:BUILD"),
+        # Unfortunately, repositories cannot depend on each other
+        # directly. They can only depend on files inside each
+        # repository. We need to be careful to depend on files that
+        # change anytime any content in a repository changes.
+        "_nixpkgs_ghc": attr.label(default = "@io_tweag_rules_haskell_ghc_nixpkgs//:bin/ghc"),
```

Comment:
Please add a comment explaining why `bin/ghc` always change, contrary to `BUILD`. It took me a few time to understand why this is correct.

Reply:
There's a comment just above. I'm not sure what more you want to see. A description of why `bin/ghc` changes?

Reply:
Perhaps that's a nitpick, but yes.